Source/Research.h

```diff
@@ -0,0 +1,71 @@
+// Research.h
+// Classes for modifying Research structures
+#ifndef _RESEARCH_H_
+#define _RESEARCH_H_
+
+enum LabType
+{
+	ltBasic			= 1,
+	ltStandard		= 2,
+	ltAdvanced		= 3
+};
+
+// "CATEGORY" values (for both items and upgrades to these items)
+enum TechCategory
+{
+	tcFree			= 0, // 0 = Free technologies (and unavailable technologies)
+	tcBasic			= 1, // 1 = Basic labratory sciences
+	tcDefense		= 2, // 2 = Defenses (GP upgrade, walls, and efficiency engineering)
+	tcPower			= 3, // 3 = Power
+	tcVehicles		= 4, // 4 = Vehicles (which ones can be built, speed upgrades, armour upgrades)
+	tcFood			= 5, // 5 = Food
+	tcMetals		= 6, // 6 = Metals gathering
+	tcWeapons		= 7, // 7 = Weapons
+	tcSpace			= 8, // 8 = Space (spaceport, observatory, launch vehicle, skydock)
+	tcMorale		= 9, // 9 = Population (happiness)
+	tcDisaster		= 10,// 10 = Disaster warning (and defense)
+	tcPopulation	= 11,// 11 = Population (health, growth)
```

Comment:
Indentation looks off for this line on GitHub. Should perhaps use spaces for justifying comments at the end of lines.